docs/application/vstools/tools/emulator-manager.md

```diff
@@ -1,152 +1,163 @@
 
-In order to test the application in a variety of environments, you need a variety of emulators. The Emulator Manager helps you easily create and manage emulator instances. Basically, the Emulator Manager allows you to generate emulator instances from a predefined platform and template. In addition, you can define the settings of the virtual device, such as skin, resolution, and hardware devices.
+Emulator Manager allows you to generate emulator instances from a predefined platform and template. Additionally, it helps you to define and configure settings such as skin, resolution, and hardware devices of your virtual device. 
 
-The main features of the Emulator Manager are:
+Emulator Manager helps you to:
 
-- Defining a new emulator instance or hardware profile
-- Editing an existing emulator instance or hardware profile
-- Deleting an emulator instance or hardware profile
-- Starting and stopping the emulator instance
+- Define a new emulator instance or hardware profile
+- Edit an existing emulator instance or hardware profile
+- Delete an emulator instance or hardware profile
+- Start or stop an emulator instance
 
+This section explains how to use Emulator Manager to run and test your application in a virtual environment:
 
-## Accessing the Emulator Manager
+1. [Launch Emulator Manager](#launch-emulator-manager)
+2. [Create emulator instance](#create-emulator-instance)
+3. [Manage emulator instance](#manage-emulator-instance)
 
-If you do not have the Emulator Manager installed, you can install it using the Visual Studio Tools for Tizen installer.
+## Launch Emulator Manager
 
-There are 2 different ways to access the Emulator Manager from Visual Studio:
+### Prerequisites
 
-- In the Visual Studio menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
+Ensure that you have installed Emulator Manager in Visual Studio. If you have not installed Emulator Manager, see [Visual Studio Tools for Tizen](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen).
 
-    **Figure: Launch the Emulator Manager in the Tools menu**
+>**Note**
+>
+>Emulator Manager for Visual Studio has the same requirement as Emulator Manager in Tizen Studio. For more information on hardware and software requirements, see [Emulator Requirements](../../tizen-studio/setup/prerequisites.md#emulator).
+
+After installing Emulator Manager from Visual Studio, you can launch it in one of the following ways: 
 
-    ![Launch the Emulator Manager in the Tools menu](media/em-vstoolbar1.png)
+- In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
```

Comment:
Don't use special characters for HTML. See markdown guide describing escape character.

Reply:
I have tried to retain the existing document style. For more information, see https://github.com/Samsung/tizen-docs/search?l=Markdown&q=%26gt%3B.
I have researched:
https://markdown-guide.readthedocs.io/en/latest/basics.html
https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown

The ampersands and angle brackets in the text will automatically be translated into HTML entities. Otherwise, Markdown thinks that they're intended to be an HTML tag.

Please advise me to proceed further with this.